primitives/core/src/traits.rs

```diff
@@ -40,9 +59,6 @@ pub trait BareCryptoStore: Send + Sync {
 		id: KeyTypeId,
 		seed: Option<&str>,
 	) -> Result<sr25519::Public, String>;
```

Comment:
We should log a separate issue to change these return types to use the new `Error` type as well.

Reply:
This is done.